src/main/java/io/cdap/delta/bigquery/BigQueryTarget.java

```diff
@@ -98,19 +106,23 @@ public void configure(Configurer configurer) {
 
   @Override
   public void initialize(DeltaTargetContext context) throws Exception {
+
     Credentials credentials = conf.getCredentials();
-    String project = conf.getProject();
+
+    if (conf.getProject() == null) {
+      return;
```

Comment:
We should ideally throw a proper error like  illegal argument exception  in such case. 

And all such checks  should be inside the method  `getProject()`  to ensure the method is null safe.

Reply:
Updated and removed